rfcs/readme.md

```diff
@@ -0,0 +1,66 @@
+# OpenUI Design System RFC
+
+The OpenUI Design System is meant to provide a global, accessible, and themable design system for the web platform by creating high-quality and accessible components that work across any web framework.
+
+The normal GitHub pull request workflow will be used to provide bug fixes and documentation improvements to existing components, however creating new components or significantly modifying existing ones by creating variations will need to have more thorough review/discussion.
+
+This RFC (request for comments) process will allow anyone to submit issues using the provided template to begin discussions and provide the ability for new components to make their way into the design system.
+
+## When to follow this process
+
+It is necessary to follow the RFC process when you are intend to create any significant changes to the OpenUI Design System. Examples of these types of changes are:
+
+* A change which provides significantly alters existing functionality, provides new functionality, or a new primative component.
+* The removal of an existing feature or functionality
+
+Another way to look at it is, if your suggested change will require a new semantic version, or breaking change to an existing feature, it will need to go through the RFC process.
+
+This RFC process is a great way to get involved in the community, suggest new ideas, and discuss them with the community and contributors. It can also be started prior to any design or functionality has been finalized.
+
+Changes that **do not** require an RFC are:
+* Changes in documentation
+* Rephrasing, reorganizing, or refactoring without any functional changes
+* Additions to existing components which are strictly new improvements, accessibility updates, or are bug fixes.
+
+## What the process is
+
+To get a change into the design system, the RFC will first be merged into the repo under the `/rfcs/active` directory. At that point, work on inclusion of the code and documentation can begin. Once the RFC is accepted it will be moved into the `/rfcs/accepted` directory to provide history on the decision process for each component.
```

Comment:
I don't follow this from a structure perspective. We currently leverage "front data" within the json to reflect whether it is a proposal or not. Granted this is a bit different for the component lib itself and so this may make sense in this context.